orvara/orbit.pyx

```diff
@@ -1075,29 +1111,52 @@ def calc_RV(Data data, Params par, Model model):
     cdef double one_d_240 = 1./240
 
     for i in range(data.nRV):
+        model.RV[i] += _calc_RV(model.sinEA[i], model.cosEA[i], model.EA[i], one_d_24,
+                                one_d_240, pi, pi_d_2, tanEAd2, sqrt1pe_div_sqrt1me, RVampl,
+                                cosarg, sinarg, ecccosarg, fabs(model.sinEA[i]), fabs(model.EA[i]))
+    # calculate the relative rv's
+    cdef int i_rel_RV = data.nTot - data.n_rel_RV
```

Comment:
conv converts the stars RV to the relative radial velocity. Is this valid for systems with more than 2 bodies?

Reply:
My guess is that is not... my derivation of this needs the semi major axis ratio to be equal to the inverse of the mass ratio.